asset/js/setup.js

```diff
@@ -9,6 +9,12 @@ function scrollToUrlAnchorHeading() {
   }
 }
 
+function flatNestedModals() {
```

Comment:
Maybe just `flattenModals()` will do? No need the word `Nested`? If really necessary, we can add a comment to clarify that this will also flatten modals within modals.